packages/headless/src/controllers/commerce/product-listing/result-list/headless-product-listing-interactive-result.ts

```diff
@@ -46,12 +44,11 @@ export function buildInteractiveResult(
       return;
     }
     wasOpened = true;
-    engine.dispatch(logProductRecommendationOpen(props.options.result));
+    // TODO: Log on click
   };
 
   const action = () => {
     logAnalyticsIfNeverOpened();
-    engine.dispatch(pushRecentResult(props.options.result));
```

Comment:
Deleted that action/file entirely, there was no associated reducer (ie: the function wasn't doing anything at all)